linera-execution/solidity/Linera.sol

```diff
@@ -196,6 +196,11 @@ library Linera {
         AccountOwner value;
     }
 
+    struct opt_ChainId {
```

Comment:
@MathieuDutSik Where these names `opt_ChainId` come from? Can we correct names like `ChainIdOption` instead?